ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -170,7 +172,7 @@ export function WorkflowsList({match, location, history}: RouteComponentProps<an
                 actionMenu: {
                     items: [
                         {
-                            title: 'Submit New Workflow',
+                            title: t('submitNewWorkflow'),
```

Comment:
Looking at this again, I feel like the translations would be easier to read/edit/write if they were 1-to-1 with the English, no? Since that's what we're translating from. The current approach effectively creates a "code word" for each translation that loses spacing and capitalization information.

i.e. if the inputs were 1-to-1:
```suggestion
                            title: t('Submit New Workflow'),
```